### PR TITLE
WIP: Mass conversion from ABAP system to files

### PR DESCRIPTION
**Pull request by albertmink**

### CODE REVIEW

File: file-formats/clas/type/zif_aff_clas_v1.intf.abap [hunk 1]
```diff
@@ -76,7 +76,7 @@ INTERFACE zif_aff_clas_v1 PUBLIC.
       message_class        TYPE c LENGTH 20,
       "! <p class="shorttext">Descriptions</p>
       "! Descriptions maintained in SE80
-      descriptions         TYPE  zif_aff_oo_types_v1=>ty_descriptions,
+      descriptions TYPE if_aff_oo_types_v1=>ty_descriptions,
```
Comment (albertmink) [inline file-formats/clas/type/zif_aff_clas_v1.intf.abap:79]: namespaces

File: file-formats/intf/examples/z_aff_example_intf.intf.json [hunk 2]
```diff
@@ -7,16 +7,10 @@
   "descriptions": {
     "types": [
       {
-        "name": "EXAMPLE_TYPE",
+        "name": "TY_EXAMPLE_TYPE",
         "description": "This is an example type"
       }
     ],
-    "attributes": [
```
Comment (albertmink) [inline file-formats/intf/examples/z_aff_example_intf.intf.json:14]: maybe we want still have attributes here as a showcase/example

File: file-formats/intf/type/zif_aff_intf_v1.intf.abap [hunk 3]
```diff
@@ -52,7 +52,7 @@ INTERFACE zif_aff_intf_v1 PUBLIC.
       proxy        TYPE abap_bool,
       "! <p class="shorttext">Descriptions</p>
       "! Descriptions maintained in SE80
-      descriptions TYPE  zif_aff_oo_types_v1=>ty_descriptions,
+      descriptions TYPE if_aff_oo_types_v1=>ty_descriptions,
```
Comment (albertmink) [inline file-formats/intf/type/zif_aff_intf_v1.intf.abap:55]: namespaces

**Comment by larshp** [comment]
is this PR still relevant? https://github.com/SAP/abap-file-formats/issues/212 was closed

**Comment by albertmink** [comment]
continued in #222

**Pull request closed without merging**